config/prod/nf-syn53689508-s3.yaml

```diff
@@ -0,0 +1,31 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.0/templates/S3/synapse-external-bucket.j2"
+stack_name: "nf-syn53689508-s3"
+stack_tags:
+  OwnerEmail: 'jineta.banerjee@sagebionetworks.org'
+  CostCenter: "NTAP NF Addendum 4 / 301100"
```

Comment:
It looks like this isn't an [active cost center](https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2553544733/Finance+Program+Codes) anymore, but `NTAP NF Amendment 5 / 301101` and `NTAP NF Amendment 6 / 301102` both are. Are either of those two cost centers appropriate here?

Reply:
@jesusaurus I found this one in the program codes: `JHU NTAP / 301100`. Can we use that one?

Reply:
@jaybee84 yeah, that works too

Reply:
done